src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -353,39 +353,42 @@
                             <CornerRadius x:Key="BottomButtonCornerRadius">0,0,4,4</CornerRadius>
                             <CornerRadius x:Key="LeftButtonCornerRadius">4,0,0,4</CornerRadius>
                             <CornerRadius x:Key="RightButtonCornerRadius">0,4,4,0</CornerRadius>
+                            <Color x:Key="GrapherAxesColor">Black</Color>
+                            <Color x:Key="GrapherBackground">White</Color>
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="Dark">
                             <CornerRadius x:Key="TopButtonCornerRadius">4,4,0,0</CornerRadius>
                             <CornerRadius x:Key="BottomButtonCornerRadius">0,0,4,4</CornerRadius>
                             <CornerRadius x:Key="LeftButtonCornerRadius">4,0,0,4</CornerRadius>
                             <CornerRadius x:Key="RightButtonCornerRadius">0,4,4,0</CornerRadius>
+                            <Color x:Key="GrapherAxesColor">White</Color>
+                            <Color x:Key="GrapherBackground">Black</Color>
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="HighContrast">
```

Comment:
>HighContrast [](start = 51, length = 12)

I'm not seeing things in the graph change when we go directly from high contrast black to high contrast white.  I'm not sure if this is our bug or a UWP bug though.

Reply:
Could be related to an issue I experienced before where the high contrast colors seem to be equal to each other when switching between the modes. I had a work around where I set them to transparent inbetween, not sure if it would help here.

Reply:
The issue was because we used `<StaticResource x:Key... />` so the color wasn't updated when we switch from a High Contrast theme to another High Contrast theme. Unfortunately `<ThemeResource x:Key` don't work but I found a workaround! Instead of setting the color in the ResourceDictionary, I create a new Style, the one being able to host a `{ThemeResource}`.

@joseartrivera This issue was different, it was because when you switch from a High Contrast mode to another one, the system only updates the Color, not the Brush, meaning GraphingControl which used the brush wasn't able to detect that the color change. This doesn't happen with other controls because they use the color of the brush we pass, not the case of GraphControl which create a copy of the color.